examples/sky-set-boost-off.py

```diff
@@ -0,0 +1,3 @@
+from pyfreshintellivent.pyfreshintellivent.FreshIntellivent import Sky
+p = Sky("Fan adress","authcode from sky-auth-ex",False)
```

Comment:
Don't need to include `False`.
```suggestion
p = Sky("Fan address","authcode from sky-auth-ex", False)
```